src/report/InfectionReport.java

```diff
@@ -59,7 +59,16 @@
     private Random random;
     private double[] factorDistances;
     private double[] distances;
-    public double[] factorLocation, factorSender, factorReceiver;
+    private double[] factorLocation, factorSender, factorReceiver;
+    // data collection
+    private int[] infectedHosts; // counts for each vaccination group the amount of people getting infected
+    private int[] spreaderHosts; // counts for each vaccination group the amount of people spreading the virus
+    private double simulationTime = 0;
+    private ArrayList<Integer> listInfectionWithinTimePeriod;
+    private final double timePeriod = 600;
```

Comment:
Das 10 minuten Intervall finde ich zu klein. Ich schaff es nicht das schön in ein Diagramm zu packen. 30min finde ich besser

Reply:
Bei 10 Minuten war ich mir etwas unsicher, ob wir das so präzise bräuchten. 30 Minuten klingt für auch gut, ich setze den Wert hoch.

Reply:
Ich glaube hier gibt es noch einen Fehler. Insgesamt haben wir jetzt nach einem run 17 30min Intervalle, was eine Party von 8,5 Stunden bedeuted. Allerdings geht die Party laut Svens Beschreibung vom Movement Model im Paper nur insgesamt 8 Stunden lang. Was ist nun richtig? Oder habe ich mich verzählt?

Reply:
Die Simulation geht 8,5 Stunden. 20:30 - 5:00 Uhr morgens. Hab's im Paper vertan. Ist korrigiert

Reply:
Der Logger funktionert in einer Weise, so dass am Ende der letzte Wert zusätzlich noch ausgegeben wird. Ich kann das ändern und prüfen, falls gewünscht.

Reply:
Wie meinst du das, dass der letzte Wert noch ausgegeben wird? Wie du in der Figure 5 von der Unity 2020 Auswertung sehen kannst, ist dort der letzte Wert einfach 0. Wie sollte denn die Grafik stattdessen aussehen?

Reply:
Das mit dem letzten Wert ausgeben bezog sich auf die Zeitintervalle für die Infektionen, für jede halbe Stunde existiert ein Wert. Am Ende der Simulation wird einfachhaltshalber der letzte Wert ausgegeben, den es zu diesem Zeitpunkt gibt.
Figure 5 benutzt andere Daten, nämlich wo sie die Hosts infiziert haben. Bei der Auswertung der Locations sind alle NodeStates enthalten und ein weiterer "Other" State, der nur getriggert wird falls er keinem Bereich zugeordnet wird.